bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useState } from 'react';
+
+// reactstrap components
+// import {
+// } from "reactstrap";
+
+//css
+import  "./css/FilterResults.css"
+
+// core components
+import IndexNavbar from "../components/Navbars/IndexNavbar.js";
+import IndexHeader from "../components/Headers/IndexHeader.js";
+import DarkFooter from "../components/Footers/DarkFooter.js";
+import ExamplesNavbar from "../components/Navbars/ExamplesNavbar.js";
+
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import List from "../components/List";
+import withListLoading from '../components/withListLoading';
+
+import image_left from '../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123021)
```suggestion
import image_left from "../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png";
```